CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Screen API Coordinator/GiniScreenAPICoordinator+Camera.swift

```diff
@@ -73,6 +72,7 @@ extension GiniScreenAPICoordinator: CameraViewControllerDelegate {
     }
 
     func cameraDidTapReviewButton(_ viewController: CameraViewController) {
+        // TODO: do we need back to review event ?
```

Comment:
[Clarification]: I think our current navigation requires that and it is worth checking it.

Reply:
I will check this in the PR for the review screen.